### PR TITLE
Fixes martial arts going away after boxing and booze

### PR DESCRIPTION
**Pull request by FlattestGuitar**
Fixes [#3895](https://github.com/ParadiseSS13/Paradise/issues/3895)

*tell me this works please*

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
You're not meant to be able to have more than one martial art at the same time; if you pick sleeping carp and want to keep it, don't get soused or put on the gloves.

**Comment by FlattestGuitar** [comment]
This doesn't make you have two at once. The `base` variable was already there, it was just incorrectly used.

Boxing gloves or getting drunk temporarily takes away your "standard" fighting style in favour of the temporary one. I don't see what the logic is behind being unable to drink.

**Comment by tigercat2000** [comment]
@Fox-McCloud Hold up. Not true.
Martial arts are designed to be split into two categories, permanent, and temporary. Shit like boxing gloves or being drunk are meant to use this, leaving more permanent styles to work. This is flat-out fixing it to work as intended.

**Comment by Fox-McCloud** [comment]
@TheDZD 

You're more familiar with martial arts datums than I.

File: code/datums/martial.dm [hunk 1]
```diff
@@ -66,12 +66,11 @@
 		H.verbs += help_verb
 	if(make_temporary)
 		temporary = 1
-	if(H.martial_art && H.martial_art.temporary)
-		if(temporary)
+	if(temporary)
+		if(H.martial_art)
```
Comment (TheDZD) [inline code/datums/martial.dm]: This should also check `&& !H.martial_art.temporary`, you don't want to set the base to something temporary, as you can end up with things like permanent boxing if you mix this with drunken brawling. The one downside is that if you equip boxing gloves, then get really drunk, you'll lose both martial arts once the drunkenness wears off unless you un-equip and re-equip the gloves, but that's preferable to permanent boxing.

Comment (FlattestGuitar) [inline code/datums/martial.dm]: That isn't actually needed. Even if `H.martial_art` isn't permanent, its `base` will be the permanent art and that's what's used under this if statement.

Comment (TheDZD) [inline code/datums/martial.dm]: Oh, looks like I just misread the line below. This is fine then.

**Pull request merged**